src/components/VmActions/Action.js

```diff
@@ -133,4 +138,42 @@ ActionButtonWraper.propTypes = {
   ...Button.propTypes,
 }
 
-export { ActionButtonWraper, MenuItemAction }
+const ActionTextWrapper = (props) => {
```

Comment:
Nit-pick: maybe better name will be `ActionMenuItemWrapper`?

Reply:
Thanks! I wasn't sure what a good name for it